charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet.flags

```diff
@@ -1,67 +1,46 @@
 {{- define "kubelet-flags" -}}
-{{- if semverCompare "< 1.10" .kubernetes.version -}}
+{{- /* TODO: this flag is PROBABLY going to be removed in 1.13  */ -}}
 --allow-privileged=true \
```

Comment:
What do we do if the flag is removed?

Reply:
When it's completely removed, the only way to schedule privileged containers is via PSP.
See https://github.com/kubernetes/kubernetes/issues/58010 I don't think it'll hit `1.13`  by the looks of things

Reply:
OK. So, if we would remove it right now then we would force the people to use PSP (backwards-incompatibly). Understood, let's remove it with 1.13.